.travis.yml

```diff
@@ -54,7 +55,7 @@ before_script:
       --prefix=$HOME/install/syslog-ng
       --with-ivykis=internal
       --with-mongoc=internal
-      --with-librabbitmq=internal
+      --with-librabbitmq=system
```

Comment:
In configure.ac you dropped the whole option, so you can just delete the line.

Reply:
You are right, if it is a go I'll update this.